src/test/java/gov/di_ipv_fraud/pages/FraudPageObject.java

```diff
@@ -214,4 +231,48 @@ public void goToResponse(String validOrInvalid) {
     public void goToVerifiableCredentialsPage() {
         title.getText();
     }
+
+    public void goToEditUserLink() {
+        editUserLink.click();
+    }
+
+    public void addHouseName(String housename) {
+        housenameField.sendKeys(housename);
+    }
+
+    public void clearHouseNumber() {
+        housenumberField.clear();
+    }
+
+    public void addHouseNumber(String housenumber) {
+        housenumberField.sendKeys(housenumber);
+        fraudCRIButton.click();
+    }
+
+    public void userAddressInJsonResponse(String testHouseName) throws JsonProcessingException {
+        String testHouseNumber = "455";
```

Comment:
Should we pass name and house number so its more reusable and were not hardcoding?

Reply:
@gds-jbentham Done